topcoffea/cfg/xsec.cfg

```diff
@@ -6,8 +6,14 @@ DYJetsToLL_M_10to50     : 22635.09
 DYJetsToLL_M_50_aMCatNLO: 6025.2
 DYJetsToLL_M_50_MLM     : 6025.2
 
+t                       : 136.02
```

Comment:
Which sample does this xsec refer to? Also, I'm not sure how I feel about such a short name, it feels like it could really easily be clobbered accidentally

Reply:
That was the single top t-channel. I'll change it to `ST_t-channel`.

Reply:
Wait, but then what does the sample on line 11 refer to?

Reply:
That was the s-channel. I see what a mess the names are now, so I'm fixing them.